### PR TITLE
Update test.sh

### PR DESCRIPTION
**Pull request by pgmillon**

### CODE REVIEW

**Comment by pgmillon** [comment]
:+1:

**Comment by jcarsique** [comment]
:+1:

**Comment by pgmillon** [comment]
:+1:

**Comment by pgmillon** [comment]
:+1:

**Comment by pgmillon** [comment]
:+1:

**Comment by pgmillon** [comment]
:+1:

**Comment by pgmillon** [comment]
:+1:

File: test.sh [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #!/bash/bin
 
+echo hello world
+
+something new
```
Comment (jcarsique) [inline test.sh:5]: test commment

Comment (jcarsique) [reply test.sh:5]: another comment

**Pull request closed without merging**